### PR TITLE
bpo-26835: Add file-sealing ops to fcntl

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
I have added  File Sealing operations to fnctl .

https://bugs.python.org/issue26835

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2019-04-29-21-33-12.bpo-26835.JYlsPS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+fcntl now has File Sealing operations.
```
Comment (pablogsal) [inline Misc/NEWS.d/next/Library/2019-04-29-21-33-12.bpo-26835.JYlsPS.rst:1]: Please, use markdown formatting for fcntl (:func:`fcntl.fcntl`).

File: Doc/whatsnew/3.8.rst [hunk 2]
```diff
@@ -181,6 +181,9 @@ Other Language Changes
   and Windows use this to properly terminate scripts in interactive sessions.
   (Contributed by Google via Gregory P. Smith in :issue:`1054041`.)
 
+* fcntl now has File Sealing operations.(Contributed by Joannah Nanjekye
```
Comment (pablogsal) [inline Doc/whatsnew/3.8.rst:184]: Please, use markdown formatting for fcntl (:func:`fcntl.fcntl`).

**Comment by pablogsal** [comment]
I think we need tests for these options, taking into account that some of them are hardcoded.

File: Doc/whatsnew/3.8.rst (hunk 2 above)
Comment (eamanu) [reply Doc/whatsnew/3.8.rst:184]: What about write the final `.` after "()" And don't write the `.` inside the parenthesis. Like this:

```
fcntl now has File Sealing operations (Contributed by Joannah Nanjekye in :issue:`26835`).
```

File: Modules/fcntlmodule.c [hunk 3]
```diff
@@ -15,6 +15,33 @@
...
+#    define F_LINUX_SPECIFIC_BASE      1024
+#endif
+
+/* fcntl.h */
+#ifndef F_ADD_SEALS
+
+/*
+ * Set/Get seals
+ */
+#define F_ADD_SEALS     (F_LINUX_SPECIFIC_BASE + 9)
```
Comment (serhiy-storchaka) [inline Modules/fcntlmodule.c:28]: Rather of defining constants with fake values, it is better to add them conditionally, as other platform-specific constants. They should not be present on non-Linux and on old Linux.

File: Doc/whatsnew/3.8.rst (hunk 2 above)
Comment (serhiy-storchaka) [reply Doc/whatsnew/3.8.rst:184]: Conform to the style used in other entries.

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]

**Comment by csabella** [comment]
Superseded by #13694.  @nanjekyejoannah, thank you for your work on this.

**Comment by nanjekyejoannah** [comment]
Thanks @tiran , Also next time it is good to ping author before opening replacement PR. It is just that some authors may feel bad for the time they Invested :)

**Pull request closed without merging**